donorsearch.php

```diff
@@ -40,7 +40,6 @@ function donorsearch_civicrm_install() {
     )),
     'permission' => 'administer CiviCRM',
   ));
-  CRM_Core_BAO_Navigation::resetNavigation();
```

Comment:
@colemanw actually I was getting a error on the next line when the child navigation create.api throw error 
```'215' is not a valid option value``` on install.  Here 215 is the parent_id reason why I have called it twice once after parent link and one after child.